cpp/src/arrow/compute/kernels/scalar_nested_test.cc

```diff
@@ -39,6 +39,46 @@ TEST(TestScalarNested, ListValueLength) {
   }
 }
 
+TEST(TestScalarNested, ListElement) {
+  for (auto ty : {list(int16()), fixed_size_list(uint8(), 2)}) {
```

Comment:
Why not test large list here as well?

Reply:
We have a test for large list bellow that one, see line 51